### PR TITLE
Added download link into the navbar

### LINKED ISSUE #347: Feature: Add download link to the nav bar
### Before you start...

- [X] Have you refreshed the page? You might be using an old version of the website.
- [X] Have you checked if someone already requested this feature? If they have, post a comment with your use case instead.

### Description

The download option is currently accessible from the home page but it's hard to discover from other pages. Adding a link to the nav bar should solve this issue.

### Alternative solutions

_No response_

### Relevant Assets

_No response_

### PR DESCRIPTION
**Pull request by oleklukasiewicz**
## Description
- Added download button into the navbar.
- Added new translations for the download button.
- Added new accent type for navbar item.

## Motivation and Context
Closes #347

## Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by yaira2** [comment]
Just a heads up that translations should only be added to the `en.json` file.

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -38,6 +39,12 @@
 			name: $_("navbar.news"),
 			path: "/blog",
 			icon: News
+		},
+		{
+			name: $_("navbar.download"),
+			path: "/download",
+			type:"accent",
+			icon: Arrow_Download
```
Comment (yaira2) [inline src/routes/+layout.svelte:47]: Is an icon needed?

**Comment by yaira2** [comment]
@oleklukasiewicz can you try moving the button to the right side of the menu? This can replace the links to the code and Discord server.
image

**Comment by oleklukasiewicz** [comment]
> @oleklukasiewicz can you try moving the button to the right side of the menu? This can replace the links to the code and Discord server. image

Sure

**Comment by oleklukasiewicz** [comment]
@yaira2 Like that?
image

**Comment by yaira2** [comment]
Let's give that a try, if anyone has a better idea down the line we can make changes then.

File: src/layout/Navbar/Navbar.svelte [hunk 2]
```diff
@@ -167,20 +178,26 @@
...
+				class:accent={type == "accent"}
+				class:selected={$page.url.pathname === path ||
+					($page.url.pathname.split("/").length > 1 &&
+						path.split("/").length > 1 &&
+						$page.url.pathname.startsWith(path) &&
+						!(path === "" || path === "/")) ||
+					(path === "/" && $page.url.pathname === "")}
+				href={path}
+				target={external ? "_blank" : undefined}
+				rel={external ? "noreferrer noopener" : undefined}
```
Comment (chingucoding) [inline src/layout/Navbar/Navbar.svelte:194]: Please convert this into a util function and document what it does. This complexity inline seems a bit too much.

File: src/i18n/locales/en.json [hunk 3]
```diff
@@ -83,8 +83,9 @@
 		"home": "Home",
 		"docs": "Documentation",
 		"news": "News",
+		"download": "Download",
 		"themes": "Themes",
 		"discord": "Discord",
 		"github": "Github"
```
Comment (chingucoding) [inline src/i18n/locales/en.json:89]: Are those two still needed?

**Review by chingucoding — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en.json (hunk 3 above)
Comment (yaira2) [reply src/i18n/locales/en.json:89]: I think they are used in the footer but didn't confirm

Comment (TheOnlyTails) [reply src/i18n/locales/en.json:89]: No, the footer uses separate translation strings.

**Review by chingucoding — DISMISSED** [review]

**Comment by yaira2** [comment]
After playing around with this, the accent color looks out place. Would it be too much trouble to switch to a regular link and put it back with the rest of the menu items? Apologies for changing my mind.

**Comment by oleklukasiewicz** [comment]
> After playing around with this, I think the accent color looks out place. Would it be too much trouble to switch to a regular link and put it back with the rest of the menu items? Apologies for changing my mind.

No problem 😉, with icon or not?

**Comment by yaira2** [comment]
With an icon

**Comment by oleklukasiewicz** [comment]
> @oleklukasiewicz this is really good! Should the link be added to the footer as well?

Sure!

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**